include/wx/valtext.h

```diff
@@ -68,31 +74,75 @@ class WXDLLIMPEXP_CORE wxTextValidator: public wxValidator
 
     wxTextEntry *GetTextEntry();
 
+    // strings & chars inclusions:
+    // ---------------------------
+
     void SetCharIncludes(const wxString& chars);
+    void AddCharIncludes(const wxString& chars);
+
     void SetIncludes(const wxArrayString& includes) { m_includes = includes; }
-    inline wxArrayString& GetIncludes() { return m_includes; }
+    void AddInclude(const wxString& include)
+    {
+        if ( !include.empty() )
```

Comment:
We should decide if passing an empty string here is an error or not. If it is, we should have a `wxCHECK_RET` instead of simply returning silently and also check for empty strings in `SetIncludes()`. If it isn't, we should just add it.

In any case it's illogical to do different things here and in `SetIncludes()`.

Reply:
I'll go with adding wxCHECK_RET because empty strings are controlled by wxFILTER_EMPTY flag